packages/wix-ui-tpa/src/components/TextArea/TextArea.tsx

```diff
@@ -78,9 +101,18 @@ export class TextArea extends React.Component<TextAreaProps> {
       autoFocus,
       className,
       maxLength,
+      withFocusRing,
     } = this.props;
     const dataObject = this._getDataAttributes();
     const showErrorIcon = error && errorDescription;
+    const { mobile: isMobile } = this.context;
+
+    const focusedBox =
+      !isMobile &&
+      withFocusRing &&
+      this.state.focused &&
+      theme === TextAreaTheme.Box;
```

Comment:
This case is different from RadioButton because we should also apply it on the "line" theme.
I think we can change to:
```suggestion
    const hasFocusRing =
      !isMobile &&
      withFocusRing &&
      this.state.focused
```